.style.yapf

```diff
@@ -5,7 +5,8 @@
 [style]
 based_on_style = yapf
 COALESCE_BRACKETS = True
+SPLIT_ALL_COMMA_SEPARATED_VALUES = True
```

Comment:
I think we can use `split_all_top_level_comma_separated_values` instead to achieve a reasonable result: https://github.com/google/yapf/issues/574

Reply:
we could, as long as we are okay with having to possible do manual cleanup when embedded args/structs get too long or messy?

Reply:
That's fine by me.